### PR TITLE
DM-45275: Failure to build prompt-service with new prompt-base 

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ARG STACK_TAG=d_latest
 FROM lsstsqre/centos:${STACK_TAG}
 ENV PYTHONUNBUFFERED True
+ARG kafka=python-confluent-kafka
 RUN source /opt/lsst/software/stack/loadLSST.bash \
```
Comment (ktlim) [inline base/Dockerfile:5]: You might want to look into the relatively new ["here docs" feature](https://docs.docker.com/reference/dockerfile/#here-documents) of Dockerfiles to improve the readability of this RUN command.  (Make sure to include `set -e` to get the effect of the `&&` command separators.)

File: base/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ARG STACK_TAG=d_latest
 FROM lsstsqre/centos:${STACK_TAG}
 ENV PYTHONUNBUFFERED True
+ARG kafka=python-confluent-kafka
```
Comment (ktlim) [inline base/Dockerfile:4]: I think it's a little unusual to use lowercase for an argument name, but it's not wrong.

**Review by ktlim — APPROVED** [review]
The logic looks correct.

File: base/Dockerfile (hunk 1 above)
Comment (kfindeisen) [reply base/Dockerfile:5]: Do you really need `set -e`? The fourth example shows the command chaining being done on the main line, which seems cleaner.

File: base/Dockerfile (hunk 2 above)
Comment (hsinfang) [reply base/Dockerfile]: Changed it to upper case.

File: base/Dockerfile (hunk 1 above)
Comment (hsinfang) [reply base/Dockerfile]: Thanks for the reference.  This is changed to use here documents.

**Pull request merged**